lib/core/datatable/components/datatable/datatable.component.html

```diff
@@ -33,7 +33,7 @@
                  [attr.aria-sort]="col.sortable ? (getAriaSort(col) | translate) : null"
                  adf-drop-zone dropTarget="header" [dropColumn]="col">
                 <span *ngIf="col.title" class="adf-datatable-cell-value">{{ col.title | translate}}</span>
-                <span class="adf-sr-only" aria-live="polite">{{ getSortLiveAnnouncement(col) | translate: {string: col.title} }}</span>
+                <span *ngIf="col.title" class="adf-sr-only" aria-live="polite">{{ getSortLiveAnnouncement(col) | translate: {string: col.title} }}</span>
```

Comment:
I think it should be based in `col.sortable`. this property defines if a column should have sortable capabilities